### ISSUE TITLE
Bluetooth devices can't be used unless using root

### ISSUE THREAD

**Issue by UriHerrera** [Nitrux/nitrux-bug-tracker#11]
**Describe the bug**
Bluetooth devices can't be initialized, and the `bluetoothd` daemon can't be started unless using root permissions.

**To Reproduce**
1. Open System Settings>Bluetooth
2. Click on Adapters
3. No adapters are found

**Expected behavior**
Users can use Bluetooth devices without root.

**Screenshots**
![image](https://user-images.githubusercontent.com/3053525/88118299-dcded700-cb82-11ea-9bff-6f0fed71a23f.png)

**Desktop (please complete the following information):**
 - Nitrux 1.3.0 OTA-21

**Additional context**
To initialize a device:
-  `hciconfig -a`
-  `sudo hciconfig hci0 up`

To start the daemon:
- `sudo bluetoothd &`

And to use the device:
- `sudo bluetoothctl`

**Comment by luis-lavaire** [comment]
We should configure a service that both detects and starts (if found) available devices.

**Comment by UriHerrera** [comment]
> We should configure a service that both detects and starts (if found) available devices.

There is a service file available, and it's running but the output of trying to run `bluetoothd`  as the user is `D-Bus setup failed: Connection ":1.156" is not allowed to own the service "org.bluez" due to security policies in the configuration file`.

**Comment by UriHerrera** [comment]
I have also tried using `setcap` to run the commands without needing to use `sudo` but it doesn't solve it.

```
sudo setcap 'cap_net_raw,cap_net_admin+eip' `which hcitool`
sudo setcap 'cap_net_raw,cap_net_admin+eip' `which hciconfig`
sudo setcap 'cap_net_raw,cap_net_admin+eip' `which /usr/lib/bluetooth/blutoothd`
sudo setcap 'cap_net_raw,cap_net_admin+eip' `which bluetoothctl`
```

**Comment by UriHerrera** [comment]
Updating `bluez` to version 5.54 doesn't change the behavior.

**Comment by anupam-git** [comment]
Try adding the user to netdev group also

**Comment by UriHerrera** [comment]
> Try adding the user to netdev group also.
> I think i read somewhere that bluetooth group is not used anymore

I added my user to that group too, still no luck.

**Comment by UriHerrera** [comment]
I have solved the problem partially by using an older package of `bluez`. I used version `5.50-1.2~deb10u1` instead of the newer 5.53 and 5.54 packages. This older version of the package makes my USB Bluetooth adapter work, however, on other computers where the Bluetooth adapter comes in a PCI-E card (Intel® Wi-Fi 6 AX200 or Intel® Dual Band Wireless-AC 8265 although both say that the Bluetooth chip is using USB) is not available as an adapter in System Settings.
